README.rst

```diff
@@ -6,6 +6,44 @@ Public Cloud Information Service enables users to lookup Public Cloud
 images and services information via REST API. Image and Server information
 is tracked in a PostgreSQL database.
 
+=============
```

Comment:
Would also be a good idea to outline the workflow for changing the schema, even if just at a high level.

Reply:
sure, I'll add some notes on that.